subsys/net/lib/lwm2m/lwm2m_obj_swmgmt.c

```diff
@@ -0,0 +1,756 @@
+/*
+ * Copyright (c) 2021 Voi Technology AB
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define LOG_MODULE_NAME net_lwm2m_obj_swmgmt
+#define LOG_LEVEL CONFIG_LWM2MM_LOG_LEVEL_DBG
```

Comment:
Typo: Should be `CONFIG_LWM2M_LOG_LEVEL_DBG`

Reply:
And I do typing for a living 🤦 thanks
I changed this to `CONFIG_LWM2M_LOG_LEVEL`. It shouldn't be the `DBG` by default.